crates/core/tests/client.rs

```diff
@@ -239,4 +190,85 @@ mod tests {
             balances
         );
     }
+
+    #[tokio::test]
+    async fn test_get_logs() {
```

Comment:
I think the test fixture `#[with(TestContext::ERC20)] kakarot_test_env_ctx: KakarotTestEnvironmentContext` can be useful here 🙂